core/domain/acl_decorators.py

```diff
@@ -72,40 +71,286 @@ def check_activity_accessible(
             activity_rights.viewable_if_private)
 
 
-def check_exploration_editable(user_id, user_actions, exploration_id):
-    """Returns a boolean to signify whether given exploration is editable
-    by the user or not.
+def check_can_edit_activity(
+        user_id, user_actions, activity_type, activity_id):
+    """Checks whether the user can edit given activity.
 
     Args:
         user_id: str. Id of the given user.
```

Comment:
Either the function's contract can accept None, or it cannot. If it can, then the docstring should be modified. If not, then it should not be, but callers should not pass None into the function.

So you need to figure out what you want the function contract to be, and have the documentation represent that faithfully.